README.md

```diff
@@ -12,75 +12,74 @@ Please note that this tool is currently in Preview, and need your feedback for i
 ```
 
-Now you can mount using the provided mount script (mount.sh):
+Create your mountpoint (/path/to/mount) and mount with blobfuse:
 ```
-./mount.sh </path/to/mount>
+blobfuse /path/to/mount --container-name=mycontainer --tmp-path=/mnt/blobfusetmp
 ```
 
+For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse/wiki/3.-Configuring-and-Running)
+
 ### Mount Options
 - You can modify the default FUSE options in mount.sh file. All options for FUSE is described in the [FUSE man page](http://manpages.ubuntu.com/manpages/xenial/man8/mount.fuse.8.html)
 - In addition to the FUSE kernel module options; blobfuse offers following options:
 	* --tmp-path=/path/to/cache : Configures the tmp location for the cache. Always configure the fastest disk (SSD or ramdisk) for best performance. 
-	* --config-path=/path/to/connection.cfg : Configures the path for the file where the account credentials are provided
-	* [OPTIONAL] --container-name=container : Required if no configuration file is specified. Also set account name and key via the environment variables AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_ACCESS_KEY
+	* --config-file=/path/to/connection.cfg : Configures the path for the file where the account credentials are provided
+	* [OPTIONAL] --container-name=container : Required if no configuration file is specified. Also set account name and key/SAS via the environment variables AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_ACCESS_KEY/AZURE_STORAGE_SAS_TOKEN
 	* [OPTIONAL] --use-https=true/false : Enables HTTPS communication with Blob storage. True by default. 
 	* [OPTIONAL] --file-cache-timeout-in-seconds=120 : Blobs will be cached in the temp folder for this many seconds. 120 seconds by default. During this time, blobfuse will not check whether the file is up to date or not.
+	* [OPTIONAL] --log-level=LOG_WARNING : Enables logs written to syslog. Allowed values are LOG_OFF|LOG_CRIT|LOG_ERR|LOG_WARNING|LOG_INFO|LOG_DEBUG
 	
+## Considerations
+
+### Design
+- When blobfuse receives an 'open' request for a file, it will block and download the entire content of the blob down to the cache location specified in --tmp-path
+- All read and writes will go to the cache location when the file is open
+- When blobfuse receives a 'close' request for the file, it will block and upload the entire content to Blob storage, and return success/failure to the 'close' call.
+- If blobfuse receives another open request within --file-cache-timeout-in-seconds, it will simply use the existing file in the local cache rather than downloading the file again from Blob storage
+
+### Performance and caching
+Please take careful note of the following points, before using blobfuse:
+- In order to achieve reasonable performance, blobfuse requires a temp directory to use as a local cache. This directory will contain the full contents of any file (blob) read to or written from through blobfuse, and will be purged as they age (--file-cache-timeout-in-seconds).
```

Comment:
This is probably a point for the wiki instead of here, but we should probably mention somewhere that the file cache timeout is not a guarantee - the cache will be neither purged nor refreshed if there are any open handles to the file, with no upper bound on the time limit